prow/plugins/label/label.go

```
@@ -70,6 +84,40 @@ type githubClient interface {
 	BotName() string
 }
 
+type AliasInterface interface {
+	hasStatusPrivilege(login string) bool
+}
+
+type Alias struct{}
+
+func (Alias) hasStatusPrivilege(login string) bool {
+	resp, err := http.Get(OWNERS_URL)
+
+	if err != nil {
+		fmt.Printf("Failed to get aliases")
+		return false
+	}
+
+	type aliasData struct {
+		// Contains the mapping between aliases and lists of members.
+		AliasMap map[string][]string `json:"aliases"`
+	}
+
+	var data aliasData
+	responseData, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		fmt.Printf("Failed to parse yaml")
+		return false
+	}
+
+	yaml.Unmarshal(responseData, &data)
```

Don't ignore the error value.